semantic.cabal

```diff
@@ -232,7 +232,6 @@ library
                      , Semantic.Git
                      , Semantic.Graph
                      , Semantic.IO
-                     , Semantic.REPL
```

Comment:
🔥—to be reborn with `semantic-analysis` & friends!

Reply:
This was a big source of compile-time slowness, so I endorse its divestment.